src/label.styles.ts

```diff
@@ -23,7 +23,7 @@ export default [
       }
 
       &.left {
-        grid-template-columns: calc(100% / 3) 1fr;
+        grid-template-columns: 33.5% 1fr;
```

Comment:
Updated guidance from Design. A breaking change? 🤔

Reply:
I'm a little torn on whether or not it's a breaking change.  Looking at the visual test report, the change is pretty minimal, nudging things over just slightly. I think a `patch` is probably fine due to that? 🤷